docs/faq.md

```diff
@@ -95,3 +95,14 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 ## How do I publish a website like this one?
 
 Check out the [**sbt-typelevel-site**](site.md) plugin.
+
+## How do I manage my ScalaJS version?
+
+We recommend explicitly setting your ScalaJS version in `project/plugins.sbt`.
+
+```
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.0")
```

Comment:
We should make this an `@SJS_VERSION@` injected variable. You can access the `scalaJSVersion` key in the build.

Reply:
And here's an example how to inject it.
https://github.com/http4s/http4s-dom/blob/17db4bf4411c62667e4f93f5e767ba61b3519bb4/build.sbt#L219-L222

Reply:
Oh wait, but the sbt-typelevel one will always be old. Hum. I guess this is fine.

Reply:
Just noting that adding
```
"SJS_VERSION" -> scalaJSVersion,
```
to https://github.com/typelevel/sbt-typelevel/blob/c810fa9f8d3d71c9fc18158537699ac7f62891e1/site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala#L114
Did not work.

But I agree that sbt-typelevel's would basically always be out of date and that might be confusing?
I wonder if we could easily pull the latest version via coursier or something.

Perhaps not worth the effort.

Reply:
Right, it can't go in the `TypelevelSitePlugin` (which doesn't depend on the `ScalaJSPlugin)` because that would roll it out to everyone's builds :) rather, this would be something specific to the sbt-typelevel `build.sbt`.

> I wonder if we could easily pull the latest version via coursier or something.

Haha temping. I might be sniped on this. I'll merge as is and then maybe give this a try.